book/07-git-tools/sections/rewriting-history.asc

```diff
@@ -8,41 +8,41 @@ Maaaring sangkot ang pagbabagot ng pagkasunod-sunod ng mga commit, pagbabago ng
 
 Sa seksyon ito, makikita mo kung paaano gagawin ang mga gawaing ito upang makagawa ka ng iyong kasaysayan na magmukha sa gusto mong unang iyong ibinahagi sa iba.
 
-[NOTE]
+[TANDAAN]
 ====
-One of the cardinal rules of Git is that, since so much work is local within your clone, you have a great deal of freedom to rewrite your history _locally_.
-However, once you push your work, it is a different story entirely, and you should consider pushed work as final unless you have good reason to change it.
-In short, you should avoid pushing your work until you're happy with it and ready to share it with the rest of the world.
+Isa sa kardinal na mga patakaran sa Git ay ang, mula ng maraming trabaho ay ang lokal na nasa loob ng iyong clone, meron kang isang magandang pakikitungo sa kalayaan upang sumulat muli ng iyong kasaysayan ng _naka-lokal_.
+Gayunpaman, Minsan ikaw ang nag-push ng iyong trabaho, ito ay isang ibang kwento sa kabuuan, at ikaw ay dapat kilalanin ang na-push na trabaho bilang huli maliban kung ikaw ay merong magandang dahilan upang baguhin ito.
+Sa madaling salita, ikaw ay dapat umiwas sa pag-push ng iyong tranaho hanggang ikaw ay masaya nito at handang ibahagi sa lahat sa mundo.
```

Comment:
> rest of the word

could also be translated to:

> sa buong mundo
_(simplified)_

Reply:
done change